File: product-service/src/main/java/com/groupproject/productservice/model/response/CatalogEntry.java
package com.groupproject.productservice.model.response;

import com.groupproject.productservice.domain.Category;
import com.groupproject.productservice.domain.Photo;
import lombok.Data;

import java.util.List;

@Data
public class CatalogEntry {

    private Long id;
    private Boolean available;
    private String brand;
    private String title;
    private String description;
    private Category category;
    private Integer price;
    private List<PropertyResponse> properties;
    private List<Photo> photos;

}

File: api-gateway/src/main/kotlin/com/groupproject/apigateway/domain/product/CatalogEntry.kt
package com.groupproject.apigateway.domain.product

data class CatalogEntry(val id: Long,
                        val available: Boolean,
                        val brand: String,
                        val title: String,
                        val description: String,
                        val category: Category,
                        val price: Float,
                        val properties: List<PropertyResponse>,
                        val photos: List<Photo>)